From: Johannes Thumshirn <jthumshirn@suse.de>
Date: Tue  7 May 09:48:57 CEST 2019
Subject: testing: nvdimm: provide SZ_4G constant
References: bsc#1132982
Patch-mainline: Never, SLE15-SP1 specific

The NV-DIMM testing harness depends on the SZ_4G constant. As discussed in
bsc#1132982 provide it locally to for the test harness.

Signed-off-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 tools/testing/nvdimm/test/nfit.c |    4 ++++
 1 file changed, 4 insertions(+)

--- a/tools/testing/nvdimm/test/nfit.c
+++ b/tools/testing/nvdimm/test/nfit.c
@@ -33,6 +33,10 @@
 
 #include <asm/mcsafe_test.h>
 
+#ifndef SZ_4G
+#define SZ_4G 0x100000000ULL
+#endif
+
 /*
  * Generate an NFIT table to describe the following topology:
  *
